File: packages/core/dto/limitrange.go
package dto

type LimitRangeRef struct {
	Namespace string `json:"namespace"`
	Name      string `json:"name"`
}

type LimitRange struct {
	Name      string
	Namespace string
	Age       string
}

type LimitRangeDetail struct {
	Name        string
	Namespace   string
	CreatedAt   string
	Age         string
	Labels      map[string]string
	Annotations map[string]string
	Limits      map[string]map[string]map[string]string
}

File: packages/core/frontend/src/types/resources/limitrange.ts
export interface LimitRange {
  Name: string;
  Namespace: string;
  Age: string;
}

export interface LimitRangeDetail {
  Name: string;
  Namespace: string;
  Age: string;
  CreatedAt: string;
  Labels: Record<string, string>;
  Annotations: Record<string, string>;
  Limits: Record<string, Record<string, Record<string, string>>>;
}
